packages/clerk-js/src/core/auth/SessionCookieService.ts

```diff
@@ -52,33 +87,24 @@ export class SessionCookieService {
   }
 
   private async refreshSessionToken(): Promise<void> {
-    if (!inBrowser()) {
-      return;
-    }
-
     if (!this.clerk.session) {
       return;
     }
 
     try {
-      this.updateSessionCookie(await this.clerk.session?.getToken());
+      await this.clerk.session.getToken();
     } catch (e) {
       return this.handleGetTokenError(e);
     }
   }
 
-  private updateSessionCookie(token: TokenResource | string | undefined | null) {
-    const rawToken = typeof token === 'string' ? token : token?.getRawString();
-
-    if (rawToken) {
-      return setSessionCookie(rawToken);
-    }
-    return removeSessionCookie();
+  private updateSessionCookie(token: string | null) {
```

Comment:
ℹ️ simplified the function signature (dropped the `TokenResource`) since it's being used internally this is not a breaking change.